types/crew.model.ts

```diff
@@ -0,0 +1,64 @@
+import { Club } from './club.model';
+import { DisplayStatus } from './display-status.model';
+import { Event } from './event.model';
+import { Result } from './result.model';
+import { Rower } from './rower.model';
+
+export interface Crew {
+	id: string;
+	club_id: string;
+	regatta_id: string;
+	event_id: string;
+	sub_event_id: null; // Wat is een sub_event?
+	subEventCode: string;
+	knrb_num: number;
+	knrb_sequence: number; // Wat is de sequence?
+	status: number;
+	result_status: null;
+	clubName: string;
+	displayName: string; // Wat is het verschil tussen 'displayName' en 'name'?
+	name: string;
+	shortname: string; // Wat is het verschil tussen 'shortname' en 'alternative'?
+	alternative: string;
+	sequence: number;
+	combination: null;
+	boat: string; // Wat is boat en boattype?
+	boattype: string;
+	remarks: null; // Wat is het verschil tussen 'remarks' en 'public_remarks'?
+	public_remarks: null;
+	remarks_for_umpire: null;
+	penalty_reason: null;
+	correction_factor: number;
+	category: string;
+	calculated_category: string; // Wat is het verschil tussen 'category' en 'calculated_category'?
```

Comment:
Category is waar de ploeg in wil starten.
calculated_category is waar de ploeg in thuis hoort.


Bijvoorbeeld

Stel je bent als ploeg 60 jaar oud dus category -> E
Maar je wil toch liever in D starten dan mag dat.
Maar je mag dan niet stiekem naar category F toe.

Dit word gebruik bij tweede import intern maar ook voor het tonen van errors/warnings